### PR TITLE
Corrected 1V/1A to give 1 ohm

### LINKED ISSUE #15647: Dividing the Quantity 1 V by 1 A gives ohm/1000 when it should be 1 ohm.
* sympy version: 1.3
* Python version: 3.6.6
* Operating System: Win10

### Description

Dividing a Quantity with dimension voltage by a Quantity with dimension current yields ohm/1000 when I expected ohm. In the SI system, 1 V/ 1 A = 1 Ω.

### What I Did

```
>>> from sympy.physics.units import Quantity, voltage, current, ohm, convert_to
>>> vs = Quantity('vs')
>>> vs.set_dimension(voltage)
>>> vs_i = Quantity('vs_i')
>>> vs_i.set_dimension(current)
>>> convert_to(vs/vs_i, ohm)
ohm/1000
```

### Further discussion
The problem is related to the kilogram workaround and the property `scale_factor`. The default scale_factor for a Quantity is 1.
```
>>> vs.scale_factor
1.0
```

The docstring for `scale_factor' states:

> Overall magnitude of the quantity as compared to the canonical units.

But, the scale factor for ohm is 1000.
```
>>> ohm.scale_factor
1000

This value of 1000 conflicts with the definition. `scale_factor` is a user facing property and should be consistent with the unit system definition, in this case the SI. The kilogram workaround should be an internal implementation factor and not exposed to the user.

### PR DESCRIPTION
**Pull request by RituRajSingh878**
#### References to other Issues or PRs

Fixes  #15647".

#### Brief description of what is fixed or changed
ohm.scale_factor=1000

This value of 1000 conflicts with the definition, this should be 1 .
and the reason of this is "scale_factor"  of "kg" is not one.
so i changed "definitions" file to make it 1.
and also test file. 

#### Release Notes

- physics.units
  - corrected scale factor Kilogram to 1 as per SI unit.

### CODE REVIEW

File: sympy/physics/units/definitions.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
 # gram; used to define its prefixed units
 g = gram = grams = Quantity("gram", abbrev="g")
 gram.set_dimension(mass)
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
 
 # NOTE: the `kilogram` has scale factor 1000. In SI, kg is a base unit, but
```
Comment (smichr) [inline sympy/physics/units/definitions.py]: Does this note still apply?

File: sympy/physics/units/definitions.py [hunk 2]
```diff
@@ -53,7 +53,7 @@
 # gram; used to define its prefixed units
 g = gram = grams = Quantity("gram", abbrev="g")
 gram.set_dimension(mass)
-gram.set_scale_factor(One)
+gram.set_scale_factor(Rational(1, 1000))
```
Comment (smichr) [inline sympy/physics/units/definitions.py]: Could this be `kg/kilo`? That would require defining kg first, however, right?

**Comment by smichr** [comment]
@RituRajSingh878 , I made a few mods. If those are agreeable to you then would you mind squashing all the commits into 1 and repushing the changes here? `git push -f RituRajSingh878 SiUnit`

File: sympy/physics/units/definitions.py (hunk 1 above)
Comment (jksuom) [reply sympy/physics/units/definitions.py]: According to [this comment](https://github.com/sympy/sympy/pull/15664#issuecomment-449357217) it seems to apply.

File: sympy/physics/units/definitions.py (hunk 2 above)
Comment (RituRajSingh878) [reply sympy/physics/units/definitions.py]: it seems right to me.

**Pull request closed without merging**